.babelrc

```diff
@@ -3,10 +3,7 @@
     "presets": [
         ["@babel/preset-env", {
             "targets": {
-                "browsers": [
-                    "last 2 versions"
-                ],
-                "node": 12
```

Comment:
No browser support?

Reply:
The commit details this a bit: https://github.com/matrix-org/matrix-js-sdk/pull/1127/commits/f1475cd3d79e0a2c536ad7014ebb3e15d759daf3

The tldr version is this babel config isn't used for the browser bundle - browserify is.

Reply:
turns out this is a partial lie, but https://github.com/matrix-org/matrix-js-sdk/pull/1127/commits/b376a7c39918c226f0b0a4a54ff3ed2e7f36c176 fixes it (see commit message)

Reply:
I guess I'm unclear on what we are saying we now export. As of b376a7c I think it's:
 * ES2015 in the browser bundle (as we're using preset-env with an empty target list)
 * Whatever node 10 supports for the npm exports, which presumably means if you're targeting node 10+ you're fine, but if you're targeting a browser you must now transpile?

What's the intention here and can we be very explicit about it?

Reply:
yea, I'll put it in the readme. The short version is:

* Browser bundle supports whatever the modern browsers do (ES2015 I think)
* ES6 for everything else
* Consumers are meant to figure out what they need/want out of this. In the case of riot-web, we're taking ES6 and bundling it ourselves.

Reply:
Should be covered by https://github.com/matrix-org/matrix-js-sdk/pull/1127/commits/89173be05504614c843654852a50e064d4c71a5d

Reply:
OK - I remember from looking at this last time that babel has no way to explicitly ask it to output a version of ES, so presumably we are relying on 'node 10' being at least approximately equal to ES6?

Reply:
yea, and in practice it's basically just adding some filler to the top and bottom of each file, so it's basically source -> output with no major differences.